client/stylesheets/_toCat.scss

```diff
@@ -5318,8 +5323,18 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileStars{
-    color: $color-purple;
+    color: gold;
     font-size: 1.5em;
+    vertical-align: middle !important;
+}
+
+.profileStars span {
+    color: #777 !important;
+    vertical-align: middle !important;
+}
+
+.profileStars span:hover {
```

Comment:
Merge rule `.profileStars span:hover` with rule on line 5325<br>Selector `profileStars` should be written in lowercase with hyphens